src/status_im2/contexts/wallet/home/view.cljs

```diff
@@ -1,38 +1,65 @@
+    [quo2.core :as quo]
+    [react-native.safe-area :as safe-area]
+    [reagent.core :as reagent]
+    [status-im2.common.home.view :as common.home]
+    [status-im2.contexts.wallet.home.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]
+    [status-im2.contexts.wallet.home.temp :as temp]))
 
-(defn wallet-temporary-navigation
+(def tabs-data
+  [{:id :assets :label (i18n/label :t/assets) :accessibility-label :assets-tab}
+   {:id :collectibles :label (i18n/label :t/collectibles) :accessibility-label :collectibles-tab}
+   {:id :activity :label (i18n/label :t/activity) :accessibility-label :activity-tab}])
+
+(defn collectibles
+  []
+  [rn/view {:style style/empty-container}
+   [rn/view {:style style/image-placeholder}]
+   [quo/text {:weight :semi-bold :style {:margin-top 12}} (i18n/label :t/no-collectibles)]
+   [quo/text {:size :paragraph-2 :style {:margin-top 2}} (i18n/label :t/no-collectibles-description)]])
+
+(defn activity
   []
-  [rn/view
-   {:style {:flex            1
-            :align-items     :center
-            :justify-content :center}}
-   [quo/text {} "TEMPORARY NAVIGATION"]
-   [quo/button {:on-press #(rf/dispatch [:navigate-to :wallet-accounts])}
-    "Navigate to Account"]
-   [quo/button {:on-press #(rf/dispatch [:navigate-to :wallet-create-account])}
-    "Create Account"]
-   [quo/button {:on-press #(rf/dispatch [:navigate-to :wallet-saved-addresses])}
-    "Saved Addresses"]
-   [quo/button {:on-press #(rf/dispatch [:navigate-to :wallet-collectibles])}
-    "Collectibles"]])
+  [rn/view {:style style/empty-container}
+   [rn/view {:style style/image-placeholder}]
+   [quo/text {:weight :semi-bold :style {:margin-top 12}} (i18n/label :t/no-activity)]
+   [quo/text {:size :paragraph-2 :style {:margin-top 2}} (i18n/label :t/no-activity-description)]])
 
 (defn view
   []
-  (let [top (safe-area/get-top)]
-    [rn/view
-     {:style {:margin-top      top
-              :flex            1
-              :align-items     :center
-              :justify-content :center}}
-     [quo/button
-      {:icon-only?      true
-       :type            :grey
-       :on-press        (fn [] (rf/dispatch [:show-bottom-sheet {:content wallet-temporary-navigation}]))
-       :container-style {:position :absolute
-                         :top      20
-                         :right    20}} :i/options]
-     [quo/text {} "New Wallet Home"]]))
+  (let [top          (safe-area/get-top)
+        selected-tab (reagent/atom (:id (first tabs-data)))]
+    (fn []
+      [rn/view
+       {:style {:margin-top top
+                :flex       1}}
+       [common.home/top-nav {:type :grey}]
+       [quo/wallet-overview temp/wallet-overview-state]
+       [rn/pressable
+        {:style         style/illustration-container
+         :on-long-press #(rf/dispatch [:show-bottom-sheet {:content temp/wallet-temporary-navigation}])}
+        [quo/text {:style {:color colors/white}} "Illustration here"]]
```

Comment:
Probably this should be a wallet graph component in empty state

Reply:
That's right @briansztamfater, fixed it 👍